testsuites/store/src/main/java/org/eclipse/rdf4j/sail/RDFStoreTest.java

```diff
@@ -718,6 +726,7 @@ public void testQueryBindings()
 	}
 
 	@Test
+	@Ignore("test is fundamentelly flawed as it uses multithreaded access on the same connection")
```

Comment:
I won't be sad to see the back of this test. It caused a lot of impossible to replicate errors in the past.

We do need a concurrent test suite for stores that support concurrent access in the future, but it needs to be replicable and solely test things that the API actually guarantees.